.changes/unreleased/Bugfix-20240910-144831.yaml

```diff
@@ -0,0 +1,3 @@
+kind: Bugfix
+body: 'BREAKING CHANGE - default value added to ownership_tag_keys field in azure integration, matches the same field in other integrations resources'
```

Comment:
```suggestion
body: 'BREAKING CHANGE - Changed the default value of field `ownership_tag_keys` in `opslevel_integration_azure_resources` to match the default of the GraphQL API (and the other infrastructure integration resource). This may cause the existing integration to revert any manual changes to `ownership_tag_keys` on the next plan/apply.'
```
```suggestion
body: 'BREAKING CHANGE - default value added to ownership_tag_keys field in azure integration, matches the same field in other integrations resources'
```